common/src/chain/block/consensus_data.rs

```diff
@@ -146,7 +146,7 @@ impl PoWData {
         let mut ret = !target;
         let mut ret1 = target;
         ret1.increment();
-        ret = ret / ret1;
+        ret = (ret / ret1)?;
```

Comment:
The `ìncrement` operation is still not checked.

Reply:
Maybe I should get rid of it altogether... Not sure how to fix this.

Reply:
I got rid of it and discovered a bug in PoW... thanks for pointing it out.